package.yaml

```diff
@@ -21,6 +21,17 @@ agents:
       - VERSION=$VERSION
     workDir: $HOME/projects/go-algorand
 
+  - name: rpm.centos8
+    dockerFilePath: docker/build/cicd.centos8.Dockerfile
+    image: algorand/go-algorand-ci-linux-centos8
+    version: scripts/configure_dev-deps.sh
```

Comment:
does this file still exist? I thought tsachi was removing it

Reply:
I cleared it's content, but kept it around so it won't create build failures.